data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1845 @@
+# Copyright (c) 2022 by Arachi-Lover
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Heliarch Investigation 1"
+	name "Transport Heliarch Agents"
+	description "Bring <bunks> Heliarch agents to <destination>."
+	minor
+	passengers 3
+	to offer
+		random < 65
+		"coalition jobs" >= 70
+		has "Coalition: Contributor: done"
+		not "Resistance: Questions: done"
+	source
+		near "Quaru" 4 10
+	destination "Ring of Friendship"
+	on offer
+		conversation
+			`A group of Heliarch agents rapidly approach you as you step onto the spaceport, looking rather hasty as they stand in front of you.`
+			`	"Looking for work, the human visitor is?" The Saryd asks.`
+			`	"To the <destination>, a means of transport we need," the Kimek follows.`
+			`	"<payment>, we can offer," the Arach finishes the proposal.`
+			choice
+				`	"Of course, I'll show you to my ship."`
+				`	"Sorry, I'm not looking for any work currently."`
+					decline
+			`	They thank you for agreeing to deliver them to the <planet>, and introduce themselves: the Saryd is called Iplora, the Kimek is Kirrie, and lastly the Arach is known as Tugroob. You help them settle in their bunks, wondering why they seemed so rushed.`
```

Comment:
```suggestion
			`	They thank you for agreeing to deliver them to <planet>, and introduce themselves; the Saryd is called Iplora, the Kimek is Kirrie, and lastly the Arach is known as Tugroob. You help them settle in their bunks, wondering why they seemed so rushed.`
```

Reply:
`<planet>` here is being replaced with "Ring of Friendship," and before replacements where the planet in question is one of the rings, a "the" is put in place so that it's always "the [Ring of something]."